src/ray/gcs/accessor.h

```diff
@@ -574,6 +574,21 @@ class NodeInfoAccessor {
   /// \param is_pubsub_server_restarted Whether pubsub server is restarted.
   virtual void AsyncResubscribe(bool is_pubsub_server_restarted) = 0;
 
+  /// Set the internal config string that will be used by all nodes started in the
+  /// cluster.
+  ///
+  /// \param config Map of config options
+  /// \return Status
+  virtual Status AsyncSetInternalConfig(
+      std::unordered_map<std::string, std::string> &config) = 0;
+
+  /// Get the internal config string from GCS.
+  ///
+  /// \param callback Processes a map of config options
+  /// \return Status
+  virtual Status AsyncGetInternalConfig(
+      const MapCallback<std::string, std::string> &callback) = 0;
```

Comment:
How about use `OptionalItemCallback<std::unordered_map<std::string, std::string>>` like line 492-501 above ?

Reply:
Works for me!